### ISSUE TITLE
White space at the bottom

### ISSUE THREAD

**Issue by billypap1** [software-mansion/react-native-screens#287]
I use react-native-screens v2.0.0-alpha.25 with react-navigation v4. When I navigate to a screen, there is a white-blank space at the bottom. I don't use SafeAreaView or anything like that. 
I hide tabbar automatically when I navigate to that screen. Is this a bug? 
My code to hide tabbar: 

```
Navigator.navigationOptions = ({ navigation }) => {

  let tabBarVisible = true;

  let routeName = navigation.state.routes[navigation.state.index].routeName

  if ( routeName == 'ThePage' ) {
      tabBarVisible = false
  }
  return {
      tabBarVisible,
  }
}
```

**Comment by AstroCorp** [comment]
It happens to me in IOS, after going back, leave a blank space on the left

**Comment by CoryWritesCode** [comment]
@billypap1 we recently found this also. It seems like there's something conflicting in the cardStyleInterpolator for the transition spec you might be using. We swapped them around and found one that worked. ( We were using `forModalPresentationIOS` with some custom adjustments I believe and swapped to using just the card style interpolator from `forFadeFromBottomAndroid` )

Not sure if it's the way react-native-screens handles the view or the way react-navigation handles the card interpolation and animation that causes it. Haven't had enough time to dive into it.

Hope this helps!
